posts/rse_at_eth/index.qmd

```diff
@@ -0,0 +1,62 @@
+
+## Lightning Talks
+
+With eight different projects and seven different research areas represented in the lightning talks, you might not think that there would be much in common between the speakers. After all, the speakers were presenting their software solutions for their highly specialised fields, ranging from the need to better control experiments in high-voltage physics through software to anonymising academic peer review.
+
+However, it was surprising to see how well the audience related to the problems presented. The audience noted the similarities in the nature of the problems faced by RSEs, such as data processing and re-entry after errors. Participants recognised the need for a local community that promotes collaboration when faced with similar local issues. This way, researchers do not have to reinvent the wheel every time they want to use software to advance their research.
+
+## Keynote - Peter Schmidt
+
+The lightning talks were followed by a speech from Peter Schmidt, who has an academic background in physics and software engineering experience. On the side, he is active in the UK RSE community and hosts a podcast called "Code for Thought", which touches upon many different RSE topics, such as reproducibility, net zero in computing, and, my favorite: [Open Data, without friction](https://codeforthought.buzzsprout.com/1326658/14165274-en-open-data-without-friction-sara-petti-and-evgeny-karev). In his talk, Peter shared the journey of the UK RSE chapter, which started in 2013 with a group smaller than ours. He shared some key resources for RSEs, such as the Software Sustainability Institute's [fellowship program](https://www.software.ac.uk/programmes/fellowship-programme) and a campaign called the [hidden REF](https://hidden-ref.org) which aims to "celebrate all research outputs", beyond publications and researchers. 
+
+## Community Building
+
+Finally, the audience was divided into groups to discuss three questions shown below. This part of the program generated a lot of interesting discussion, which led to the following ideas worth sharing: 
+
+- Q1: What are the expectations of the community, and how do we get there (short and long-term)?
+    * Long-term: seek SNF funding for joint RSE projects from different labs
+    * Launch a mentoring program 
+
+- Q2: How do we increase visibility, engagement, and diversity?
+    * Create ambassador roles within institutes
+    * Seek industry representation 
+
+- Q3: Should we form special interest groups?
+    * Yes, once we reach a certain size: group by interest/technology rather than by discipline
+    * the overall aim is not to fragment the community
+
+Given the lively discussions and the motivation felt in the audience, we hope to be able to implement these ideas as the RSE@ETH community grows.
```

Comment:
"we" is the wrong subject. Either: we're optimistic or "community leaders", "the RSE community" but we is rather RSEED.